src/pages/Home/HomePage.tsx

```diff
@@ -16,7 +29,11 @@ export const HomePage = (): JSX.Element => {
         <HistorySection />
         <Divider />
         <GoalSection />
-        <Carousel />
+        <StyledCarousel
+          showSingleSlide={platform !== Platform.Web}
+          imageList={IMAGE_LIST}
```

Comment:
I think you should not pass IMAGE_LIST. Instead, set 'children' and keep the 'data.map' loop here. The Carousel library should animate whatever is in 'children,' whether it's an image list, a div, or cards for blogs, reviews, or other sections.